### DISCUSSION TITLE
Sensor With Light Source (For Cameras Without Removable Back)

### DISCUSSION THREAD

**Discussion by ic-racer** [srozum/film_camera_tester#55] [Ideas]
I had been thinking about designing a sensor with a built-in light source. I don't have any old Leicas, but I do have quite a few Minox and 16mm still cameras. These are somewhat of a challenge with the sensor designs available here. 

I did realize that, without the green filter, the eyepiece view through my Autocollimator is quite bright. In fact the eyepiece concentrates the Siemens star pattern to a bright speck, about 1mm in diameter, right in front of the eyepiece. 

I found that by carefully placing one of the sensors in the correct location on the eyepiece, there was plenty of light to trigger the shutter tester with the light reflected off the film plane and out the Autocollimator's eyepiece. 
![DSC_0666](https://github.com/user-attachments/assets/db1b40d4-6ad9-47e7-8efe-a4e66ddc71ab)
![DSC_0663](https://github.com/user-attachments/assets/ada1ff70-a831-47e5-9d3a-1abc223f3bb7)

**Comment by ic-racer** [comment]
A reflector was fashioned from some polished brass and this is placed in the film gate of the 16mm camera. 

![DSC_0664](https://github.com/user-attachments/assets/b8244942-6045-452f-9faf-8b7515f506db)

**Comment by ic-racer** [comment]
![DSC_0665](https://github.com/user-attachments/assets/e8d69e9c-f471-408a-9619-ed2931e0def4)

**Comment by ic-racer** [comment]
Maybe someone reading this can come up with a STL file for one of the sensors. One that snaps on the sensor and places the 0.8mm hole exactly in the middle of a 35mm diameter slip on adapter for the eyepiece.

**Reply by srozum** [reply]
Ah, you just need clip-on adapter to mount it on collimator eyepiece.

**Comment by srozum** [comment]
That's very clever solution!

Yes, reflective metering is a totally legit method. In fact, not only Leicas, but digital cameras tested in such way. Tsubosaka still makes reflective shutter testers for digital cameras. I believe they use infra red light.

I had reflective sensor in mind and even have reserved ID (resistor value for it). But ran out of memory on Arduino to implement it, then ran out of resources etc. I am glad that Matt (Reveni Tester) added it to his sensor.

For simple shutters like Minox or other 16mm cameras or even movie cameras, adding LED (it could be standard or IR) right next to the sensor opening should work without any firmware modification. If someone could design PCB and/or STL it would be great. I think LED must have some aperture or baffle in front of it to limit and direct the light beam. I haven't done any tests in that direction, only research.

**Comment by ic-racer** [comment]
I downloaded FreeCad and tried to make the adapter part. Seemed to go OK. I'd like to attach the part to this post, in case anyone else wants something similar. Also so someone could check it before I send it to get made. I have attached a screen shot of the part. 

 Maybe I need to convert it to some other format to share it here. 

<img width="964" alt="Screen Shot 2024-11-17 at 8 38 04 AM" src="https://github.com/user-attachments/assets/277c0175-4a92-446a-85e8-75c6d8e135e4">

**Comment by ic-racer** [comment]
I was able to have my library print the part for $3. 

Now I can use my collimator for shutter speed testing. 



![DSC_0873](https://github.com/user-attachments/assets/72cb4d82-0f33-427c-b01c-c3dde25ce675)

**Reply by RobC62** [reply]
super upgrade!
What model/make of collimator do you have?
